apps/gpc-prover-client/next.config.js

```diff
@@ -1,11 +1,18 @@
 /** @type {import('next').NextConfig} */
+
 module.exports = {
   transpilePackages: ["@repo/ui"],
   webpack: (config, { isServer }) => {
     if (!isServer) {
       config.resolve.fallback = {
         fs: false
       };
+      config.resolve.alias = {
```

Comment:
Can you document what this is required for?  Maybe a comment here would be a good place to document the underlying issue.

Reply:
added comments